src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,117 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+type Props = {
+  onAdd: (movie: Movie) => void
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [title, setTitle] = useState('');
```

Comment:
you can use on state
```suggestion
  const [value, setValue] = useState({title: '', description: '', ...});
```